### PR TITLE
chore: update gno version

### PR DESCRIPTION
**Pull request by zivkovicmilos**
## Description

This PR updates the latest version of `gno` to include the API fixes on the monorepo

### CODE REVIEW

File: fetch/worker.go [hunk 1]
```diff
@@ -79,7 +80,10 @@ func getBlocksFromBatch(chunkRange chunkRange, client Client) ([]*types.Block, e
 	}
 
 	// Get the block results
-	blocksRaw, err := batch.Execute()
+	ctx, cancelFn := context.WithTimeout(context.Background(), 5*time.Second)
```
Comment (ajnavarro) [inline fetch/worker.go:83]: shall we make this configurable? If you prefer to do that on another PR, maybe we should then keep this context as a `context.Background()` to keep the previous behavior.

**Review by ajnavarro — CHANGES_REQUESTED** [review]

File: fetch/worker.go (hunk 1 above)
Comment (zivkovicmilos) [reply fetch/worker.go]: You're right, we should probably make this configurable 🤦‍♂️ 

Swapped:
[46fb3cb](https://github.com/gnolang/tx-indexer/pull/41/commits/46fb3cb89458580fecd40c58f4b6db2ae4459b32)

**Review by ajnavarro — APPROVED** [review]
